src/simsopt/util/permanent_magnet_helper_functions.py

```diff
@@ -310,12 +310,14 @@ def initialize_coils(config_flag, TEST_DIR, s, out_dir=''):
         R1 = s.get_rc(1, 0) * 2
         order = 5
 
-        # qh needs to be scaled to 0.1 T on-axis magnetic field strength
+        # QH reactor scale needs to be 5.7 T average magnetic field strength
         from simsopt.mhd.vmec import Vmec
         vmec_file = 'wout_LandremanPaul2021_QH_reactorScale_lowres_reference.nc'
-        total_current = Vmec(TEST_DIR / vmec_file).external_current() / (2 * s.nfp) / 8.75 / 5.69674966667
-        base_curves = create_equally_spaced_curves(ncoils, s.nfp, stellsym=True, R0=R0, R1=R1, order=order, numquadpoints=128)
-        base_currents = [(Current(total_current / ncoils * 1e-5) * 1e5) for _ in range(ncoils-1)]
+        total_current = Vmec(TEST_DIR / vmec_file).external_current() / (2 * s.nfp) / 1.311753
```

Comment:
what's going on here? can we add a quick comment? + update the docstring to say what the mean modB on major radius is being set here?

Reply:
Removed, and replaced by a fixed number that gets the scaling right. Updated the docstring